consensus/polybft/proto/transport.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package v1;
+
+option go_package = "/consensus/polybft/proto";
+
+message TransportMessage {
+  bytes data = 1;
+}
```

Comment:
Why do we need `TransportMessage`, when `GossipMessage` is defined, and they're literally the same structure?

Reply:
Yes currently they are the same, one is for the bridge and the other is for the consensus engine.

Reply:
Yes, currently they are the same, one is for the bridge and the other is for the consensus engine.